docs/imagecustomizer/how-to/live-iso.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Create LiveOS ISO
+parent: How To
+nav_order: 9
+---
+
+# Create a LiveOS ISO and run it in a VM
+
+This guide demonstrates how to use Image Customizer to build an Azure Linux LiveOS ISO
+and then it in a local VM.
```

Comment:
"... and then **use** it in ..."? Or launch / boot?

Reply:
Fixed.